src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/Server/sqlser.cs

```diff
@@ -340,7 +292,7 @@ public override int Read(byte[] buffer, int offset, int count)
 
         public override void Write(byte[] buffer, int offset, int count)
         {
-            this.m_size += count;
+            _size += count;
```

Comment:
Could use lambda here too.

Reply:
Changed. There are other places where it can be used but eventually looking through these files you get snowblind for some things. When the files are merged it'll get another pass so more can be cleaned up.

It will also depend on the style policy you want to apply for this repo. There are many versions of language constructs we can possibly use like switch expressions, expression bodies, static lambdas, type switching. And then the level of clarity we feel each provides. I'm being conservative at the moment.